shell/platform/darwin/common/framework/Source/FlutterNSBundleUtils.mm

```diff
@@ -26,11 +26,38 @@
   return nil;
 }
 
-NSBundle* FLTFrameworkBundleWithIdentifier(NSString* bundleID) {
-  NSBundle* bundle = FLTFrameworkBundleInternal(bundleID, NSBundle.mainBundle.privateFrameworksURL);
-  if (bundle != nil) {
-    return bundle;
+NSBundle* FLTGetApplicationBundle() {
+  NSBundle* mainBundle = [NSBundle mainBundle];
+  // App extension bundle is in Runner.app/PlugIns/Extension.appex.
```

Comment:
Nit: It's not necessarily "Runner.app", right? Maybe `<AppName>.app/PlugiIns/Extension.appex`?